tasks/post-to-cmr/tests/cmr_test.js

```diff
@@ -80,6 +80,36 @@ test.serial('should succeed with correct payload', (t) => {
     }));
 });
 
+test.serial('should succeed with correct payload and return SIT url', (t) => {
+  process.env.CMR_ENVIRONMENT = 'SIT';
+  const newPayload = JSON.parse(JSON.stringify(payload));
+  sinon.stub(cmrjs.CMR.prototype, 'ingestGranule').callsFake(() => ({
+    result
+  }));
+  const granuleId = newPayload.input.granules[0].granuleId;
+  const key = `${granuleId}.cmr.xml`;
+
+  return aws.promiseS3Upload({
```

Comment:
In general, we tend to use `async`/`await` instead of chaining Promises, since it makes the code a little easier to follow.